src/popup.tsx

```
@@ -908,6 +1016,22 @@ const PopupDefaultExample = ({
+                          style={{
+                            ...smallButtonStyle,
+                            color:
+                              sharingLabel === a.label
+                                ? THEME.primary
+                                : smallButtonStyle.color,
+                          }}
+                          onClick={() => toggleSharing(a.label)}
+                          disabled={accountsBusy}
+                          title="Share the anchor's CLAUDE.md / skills / commands with this account"
+                        >
+                          Sharing {sharingLabel === a.label ? '▾' : '▸'}
```

ESLint

[error] Replace `··························` with `······························`

(prettier/prettier)